media/css/mozorg/manifesto.scss

```diff
@@ -0,0 +1,152 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+@import '../pebbles/components/highlight';
+@import '../pebbles/components/newsletter';
+@import '../hubs/sections';
+@import '../hubs/buttons';
+@import '../hubs/masthead';
+
+.principle {
+    box-sizing: border-box;
+    margin: 0 auto;
+    max-width: $width-max-content;
+    padding: $mobile-content-padding;
+    padding-left: 2em;
+}
+
+.principle-number {
+    color: #999;
+    display: inline-block;
+    margin-left: -2.2em;
+    text-align: right;
+    width: 2em;
+    text-decoration: none;
+
+    &:visited {
+        color: #999;
+    }
+}
+
+.principle-head h3 {
+    @include font-size-level2;
+    font-weight: normal;
+}
+
+.principle-more {
+    @include open-sans;
+    @include font-size-small;
+    margin-top: $mobile-content-padding;
+}
+
+@media #{$mq-tablet} {
+    .principle {
+        padding: 0 $tablet-content-padding $tablet-content-padding $tablet-content-padding;
+    }
+}
+
+@media #{$mq-desktop} {
+    .principle {
+        padding: 0 $desktop-content-padding $desktop-content-padding $desktop-content-padding;
+    }
+}
+
+@supports (display: grid) {
```

Comment:
Not a blocker as the content is still perfectly readable in browsers that don't support CSS grid, but would it be worth considering a fallback to something like Flexbox for slightly older browsers?

Reply:
Not for a design that's only going to live a week ;)

Reply:
Ah, I wasn’t aware the design was going to change drastically again. Not worth spending too much time over 👍